scripts/build

```diff
@@ -6,5 +6,8 @@ PARENT_PATH=$(dirname $(cd $(dirname $0); pwd -P))
 
 pushd $PARENT_PATH
 mkdir -p build
-GOOS=$1 GOARCH=$2 go build -o build/snetd-$1-$2 snetd/main.go
+now=$(date +'%Y-%m-%d_%T')
```

Comment:
How is the version incremented?

Reply:
version tags will be added before this script is run ( @vinthedark  and I discussed this as part of the build)

Reply:
@anandrgitnirman , please update documentation to mirror new parameter of the build script